DOCS-11355 Gmail 1.0 (#1542)
mulesoft/docs-connectors: gmail/1.0/modules/ROOT/pages/index.adoc

= Gmail Connector - Mule 4

Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]

> Anypoint Connector for Gmail (Gmail Connector) allows for management of your Gmail account as well as sending emails and reacting to received emails.

Reviewer: ```suggestion
Anypoint Connector for Gmail (Gmail Connector) enables the management of your Gmail account, including sending and receiving emails and reacting to received emails.
```

For information about compatibility and fixed issues, see the Gmail Connector Release Notes.

== Prerequisites

To use this connector, you must be familiar with:

* The Gmail Connector’s API
* Anypoint Connectors
* Mule runtime engine (Mule)
* Elements and global elements in a Mule flow
* How to create a Mule app using Anypoint Studio

Before creating an app, you must have:

* Credentials to access the Gmail target resource
* Anypoint Platform

> * Anypoint Studio version 7.0 or later

Reviewer: Check on the Studio version. It might be 7.3 and later
